ex1/exceptions.py

```diff
@@ -37,7 +37,9 @@
 # Use the function search_n from the functions module inside a new function
 # also named search_n. The function should do the same as functions.search_n
 # but if the variable is not found in the list then raise a ValueError.
-
+def search_n(list,x):
+	return(list.index(x), x)
```

Comment:
This is arguably more elegant than what I asked for. But the main point was to work with functions that have the same name but are in different namespaces.